srv/GetVersion.srv

```diff
@@ -0,0 +1,3 @@
+---
+VersionInformation version
+bool success
```

Comment:
what would make this fail?

Reply:
Nothing, so i will remove the success parameter.